src/Command/SiteDbImportCommand.php

```diff
@@ -225,5 +225,32 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     else {
       throw new SiteCommandException($shellProcess->getOutput());
     }
+
```

Comment:
Why not putting this block after line 150, or maybe create a function and call it from line 151?